WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsUiUtils.kt

```diff
@@ -0,0 +1,28 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import org.wordpress.android.util.extensions.getString
+
+internal object BloggingPromptsUiUtils {
+    fun ImageView.showImage(@DrawableRes resId: Int) {
+        visible()
+        setImageResource(resId)
+    }
+
+    fun TextView.showText(@StringRes resId: Int) {
+        visible()
+        text = getString(resId)
+    }
+
+    fun View.visible() {
+        this.visibility = View.VISIBLE
```

Comment:
I really like the idea of changing the View visibility using extension functions, but it seems that we already have one that does the same thing in `ViewExtensions.kt` (`fun View.setVisible(visible: Boolean)`). Would it make sense to use the existing one instead?

Reply:
Ah. Yes, if it exists, let's keep it consistent to the rest of code. Will make the changes.

(I personally prefer the readability of `.gone()` and `.visible()` over `setVisible()`. But that's a change to propose at team level, not for this single feature task).

Reply:
Fixed.

Reply:
> I personally prefer the readability of .gone() and .visible() over setVisible()

  Me too 👍 

> But that's a change to propose at team level, not for this single feature task

  Yes, I agree